multiversx_sdk/network_providers/http_resources.py

```diff
@@ -189,14 +199,24 @@ def smart_contract_result_from_proxy_response(raw_response: dict[str, Any]) -> S
 
 
 def _smart_contract_result_from_response(raw_response: dict[str, Any]) -> SmartContractResult:
-    sender = raw_response.get("sender", "")
-    receiver = raw_response.get("receiver", "")
+    sender = raw_response.get("sender", "") or None
```

Comment:
I would extract a method with this code.. it looks like is in multiple places
 receiver = raw_response.get("receiver", "") or None
```
    if not receiver:
        receiver = EmptyAddress()
    else:
        receiver = Address.new_from_bech32(receiver)
```

Reply:
refactored it a little bit